src/Microsoft.PowerShell.Commands.Utility/commands/utility/write.cs

```diff
@@ -199,35 +195,33 @@ protected override void BeginProcessing()
         }
 
         /// <summary>
-        /// This method implements the ProcessRecord method for Write-Information command
+        /// This method implements the ProcessRecord method for Write-Information command.
         /// </summary>
         protected override void ProcessRecord()
         {
             WriteInformation(MessageData, Tags);
         }
 
-    }//WriteInformationCommand
+    }
 
     #endregion WriteInformationCommand
 
     #region WriteOrThrowErrorCommand
 
     /// <summary>
-    /// This class implements the Write-Error command
+    /// This class implements the Write-Error command.
     /// </summary>
     public class WriteOrThrowErrorCommand : PSCmdlet
     {
         /// <summary>
-        /// ErrorRecord.Exception -- if not specified, ErrorRecord.Exception is
-        /// System.Exception.
+        /// ErrorRecord.Exception -- if not specified, ErrorRecord.Exception is System.Exception.
         /// </summary>
         [Parameter(ParameterSetName = "WithException", Mandatory = true)]
         public Exception Exception { get; set; } = null;
 
         /// <summary>
         /// If Exception is specified, this is ErrorRecord.ErrorDetails.Message.
-        /// Otherwise, the Exception is System.Exception, and this is
-        /// Exception.Message.
+        /// Otherwise, the Exception is System.Exception, and this is Exception.Message.
```

Comment:
Suggest: I believe these two lines should be one sentence..

...this is ErrorRecord.ErrorDetails.Message; otherwise, the

Reply:
Fixed.